packages/table/src/Table.tsx

```diff
@@ -629,7 +629,7 @@ const ProTable = <T extends Record<string, any>, U extends ParamsType, ValueType
     ...props.editable,
     tableName: props.name,
     getRowKey,
-    childrenColumnName: props.expandable?.childrenColumnName,
+    childrenColumnName: props.expandable?.childrenColumnName ?? 'children',
```

Comment:
This change makes `editableUtils` consistent with [`editableDataSource`](https://github.com/ant-design/pro-components/blob/master/packages/table/src/Table.tsx#L156)